Library/Homebrew/cask/cmd/info.rb

```diff
@@ -25,52 +26,62 @@ def self.help
         "displays information about the given Cask"
       end
 
+      def self.get_info(cask)
+        "#{title_info(cask)}\n"\
```

Comment:
Use `<~EOS` which allows you to embed the newlines.

Reply:
Done! Also, do you know what I might need to do to get the CI to pass? The only test failure I can replicate locally is `brew tests --generic --online`, but `brew tests --online` passes. I'm not sure why the test would be different with the `--generic` flag.